### PR TITLE
Fixed version numbers in conda file

### PR DESCRIPTION
**Pull request by caspargross**
This pull request should solve some remaining problems to prepare the pipeline for official release:

- [x] Fix conda version numbers
- [x] Fix docker repo name

Remaining problems:
- [ ] Linting errors:  `Conda dependency had unknown version` for several tools that are valid (i.e. `r=3.5.0`

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).

**Learn more about contributing:** https://github.com/nf-core/denovohybrid/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by caspargross** [comment]
... this is my first pull request, please correct me if i did something stupid :D

**Comment by drpatelh** [comment]
All looks good in terms of PR :+1: 

Im wondering whether you can simplify `environment.yml` even further. In theory, you should be able to create the same environment with an extensive list of packages but in practice that may not happen due to low-level updates. 

e.g. just provide top-level packages required for the pipeline:
https://github.com/nf-core/denovohybrid/blob/374b6518c6b816da6776a1950463907bee8c78a9/main.nf#L152-L165

and then let `Conda` take care of the dependencies for you? Will also be much easier to see the main software dependencies for the pipeline. Once defined, you will need to tests this environment locally with the pipeline first because the `dev` version of the `Docker` container for the Travis CI tests wont be updated until this PR is merged.

May even get the linting tests to pass :wink:

File: environment.yml
Comment (drpatelh) [inline environment.yml]: Looks much better :+1: 

What does this line do? Never seen it before in an `environment.yml`.

Comment (drpatelh) [reply environment.yml]: Also, did you try and build this environment locally and test that it works with the pipeline using `-profile conda`?

Comment (caspargross) [reply environment.yml]: Removed the line, this was a leftover from trying to fix the linting issues where exoisting conda versions are not correctly identified (https://travis-ci.com/nf-core/denovohybrid/jobs/226690048#L1023).

I also added back a few tools that i removed but were needed

File: environment.yml
Comment (drpatelh) [inline environment.yml]:
```suggestion
  - r-base=3.5.0
```

Comment (drpatelh) [reply environment.yml]: I think this is why the linting is failing.

**Comment by drpatelh** [comment]
Dammit! `conda-forge` doesnt have `r-base=3.5.0`. May need to change it to `r-base=3.5.1` and possibly test if the environment still builds locally. Probably also worth running the linting locally this time to see if it works.

`conda search r-base -c conda-forge`

```
# Name                       Version           Build  Channel             
r-base                         3.3.1               1  pkgs/r              
r-base                         3.3.1               2  pkgs/r              
r-base                         3.3.1               3  pkgs/r              
r-base                         3.3.1               5  pkgs/r              
r-base                         3.3.1               6  pkgs/r              
r-base                         3.3.2               0  pkgs/r              
r-base                         3.3.2               1  pkgs/r              
r-base                         3.3.2               2  conda-forge         
r-base                         3.3.2               3  conda-forge         
r-base                         3.3.2               4  conda-forge         
r-base                         3.3.2               5  conda-forge         
r-base                         3.4.1               0  conda-forge         
r-base                         3.4.1               0  pkgs/r              
r-base                         3.4.1               1  conda-forge         
r-base                         3.4.1               1  pkgs/r              
r-base                         3.4.1               2  conda-forge         
r-base                         3.4.1               3  conda-forge         
r-base                         3.4.1               4  conda-forge         
r-base                         3.4.1      h4fe35fd_8  conda-forge         
r-base                         3.4.2      haf99962_0  pkgs/r              
r-base                         3.4.3      h1c2f66e_4  pkgs/r              
r-base                         3.4.3      h1e0a451_2  pkgs/r              
r-base                         3.4.3      h290ecf8_0  pkgs/r              
r-base                         3.4.3      h290ecf8_1  pkgs/r              
r-base                         3.4.3      h9bb98a2_5  pkgs/r              
r-base                         3.5.0      h1c2f66e_1  pkgs/r              
r-base                         3.5.0      h1e0a451_1  pkgs/r              
r-base                         3.5.1   h08e1455_1008  conda-forge         
r-base                         3.5.1      h1e0a451_2  pkgs/r              
r-base                         3.5.1   h271c98b_1006  conda-forge         
r-base                         3.5.1      h391c2eb_4  conda-forge         
r-base                         3.5.1      h391c2eb_5  conda-forge         
r-base                         3.5.1   hbc19587_1011  conda-forge         
r-base                         3.5.1   he45234b_1004  conda-forge         
r-base                         3.5.1   he45234b_1005  conda-forge         
r-base                         3.5.1   hea2b9be_1007  conda-forge         
r-base                         3.5.1   hfb2a302_1009  conda-forge         
r-base                         3.5.1   hfb2a302_1010  conda-forge         
r-base                         3.6.0      hce969dd_0  pkgs/r              
r-base                         3.6.1      h6e652e1_3  conda-forge         
r-base                         3.6.1      h8900bf8_0  conda-forge         
r-base                         3.6.1      h8900bf8_1  conda-forge         
r-base                         3.6.1      h8900bf8_2  conda-forge         
r-base                         3.6.1      hba50c9b_4  conda-forge         
r-base                         3.6.1      hce969dd_0  pkgs/r 
```

**Comment by caspargross** [comment]
I dont really need R for the analysis. Its just included to convert the markdown required by the nf-core template... 
Wouldnt it be easier to use a python library or pandoc for this anyway?

**Comment by drpatelh** [comment]
Woooohooooo! The tests have passed :smile: 

Since you have `r-markdown` in your environment its recommended by the maintainers of `Bioconda` to explicitly specify the `r-base` version you would like. Mainly to speed up resolving and building the environment.

Once this PR is merged into `dev` it will trigger the `Docker` container to be rebuilt with this new `environment.yml`. Hopefully, this builds fine and then Travis CI will automatically pull this for any new PRs you make.

Im going to approve the PR now so feel free to merge if you are happy too :+1:

**Review by drpatelh — APPROVED** [review]

**Comment by drpatelh** [comment]
> Wouldnt it be easier to use a python library or pandoc for this anyway?

Possibly, yes, seems like a waste to have an R install for a single file format conversion. Maybe this is something to suggest on the `tools` channel if youre happy to update the template or you could create an issue on `nf-core/tools` `:+1:

**Pull request merged**
